spring-web-modules/spring-mvc-basics-5/src/main/java/com/baeldung/requestparam/UserDTO.java

```diff
@@ -0,0 +1,24 @@
+package com.baeldung.requestparam;
+
+import java.util.List;
+
+public class UserDTO {
```

Comment:
We follow the Google Java style guide camel case convention for naming:
https://google.github.io/styleguide/javaguide.html#s5.3-camel-case
```suggestion
public class UserDto {
```

Reply:
OK, I just found some other classes named this.